scripts/flashing/esp32_firmware_utils.py

```diff
@@ -445,6 +445,40 @@ def actions(self):
 
         return self
 
+### Mobly integration
+class ESP32Platform:
+  def __init__(self, flasher_args):
+      self.flasher = Flasher(**flasher_args)
+
+  def flash(self):
+      self.flasher.flash_command([os.getcwd()])
+
+def verify_platform_args(platform_args):
+    required_args = [
+        'application',
+        'parttool',
+        'port',
+        'baud',
+        'before',
+        'after',
+        'flash_mode',
+        'flash_freq',
+        'flash_size',
+        'compress',
+        'bootloader',
+        'partition',
+        'partition_offset',
+        'application_offset',
+    ]
+    for r in required_args:
```

Comment:
suggest:

if any(arg not in platform_args for arg in required_args):

or

if set(required_args) - set(platform_args):

Reply:
done